src/NUnitFramework/framework/Attributes/RandomAttribute.cs

```diff
@@ -42,6 +43,11 @@ public class RandomAttribute : DataAttribute, IParameterDataSource
         private RandomDataSource _source;
         private int _count;
 
+        /// <summary>
+        /// If true, no value will be repeated
```

Comment:
I had looked at other source files and they didn't have a period in the end of the sentences, so I opted not to write it, but I'll write it.